### ISSUE TITLE
Auto Analysis makes unexpected decision. 

### ISSUE THREAD

**Issue by JuliaNaumetsss** [reportportal/reportportal#2368]
One of test in the project is failed. Errors were the following:
![image](https://github.com/user-attachments/assets/fd23777e-cc12-474b-9998-7d7f8f8130b6)

This failed test was analyzed automatically and linked to the issue (wrong issue). I have opened 'History Actions' tab and navigated to the  'based on item' link
![image](https://github.com/user-attachments/assets/88b6fa9b-7911-4f2a-8251-ff4bcad8abfd)

And I see the following errors in the second test:
![image](https://github.com/user-attachments/assets/062b4bf1-ab51-4345-9685-443b86c31f0a)
![image](https://github.com/user-attachments/assets/146428dd-87d1-4eb7-a16e-63a9e2188759)

Common between Test1 and Test2 is only 'Attempts to Log out' step
![image](https://github.com/user-attachments/assets/459dc3f5-551f-4591-8f90-37a09dd8821e) 
![image](https://github.com/user-attachments/assets/ae8cefbc-46c8-498a-8bb7-2e3a64287094)

So, it's not what we expected. Could you clarify the following questions:
1) Am I understand correctly that this behavior is expected, because [When a test item has several error logs, we will use the log with the highest score as a representative of this test item](https://reportportal.io/docs/analysis/MLSuggestions)? 
2) What is considered as 'error log' in this case:
![image](https://github.com/user-attachments/assets/ea240660-48c3-4f77-986a-b3dd3690d3c5)
And how many log errors do we have in the screenshot above: 1, 3 or 4?  
3) Is it possible to teach Autoanalysis to ignore some errors during auto-analysis? So, if test is failed with 2 errors, is it possible to ignore 1 of them (not whole test, but specific error in the test)?
For example, our tests is written in such way that they have the same clean up for each test (logout from Product). So, if this step is failed, it's displayed in the error logs and marks with high score. But we want to ignore 'log out' step from Autonalysis if it's not the only one failed step in the test.
4) Is it possible to set score for errors manually? Or any solution how to change score of error log? 

And one more question:
1) We use retry functionality. Does auto analysis take logs from second failure only? Is it possible change configuration so first/both runs will be taken into account? 
![image](https://github.com/user-attachments/assets/ded1d555-d94c-419f-8400-8a9bb131d372)

**Comment by HardNorth** [comment]
@JuliaNaumetsss which version of RP do you use?

**Comment by JuliaNaumetsss** [comment]
Version 5.7.4
![image](https://github.com/user-attachments/assets/0a2aae0d-2678-4cc7-bbdb-28ce4fd74c8c)

**Comment by HardNorth** [comment]
@JuliaNaumetsss Well, you need to update. The latest version has the best hit ratio. Also it will perform best if you group meaningful errors into one log entry, but avoid any garbage in that entry.

Basically, Analyser picks just error logs in a test item, and analyse them one-by-one. If it found the best match it put it as the result. It joins too small logs into one entry, but the efficiency of such algorithm is still under the question. So it's always better for user to skip logging insignificant data.

As I see you have many ERROR log entries in your test. It's better to have one, even with some garbage, but not too huge. Basically ML means "crap-in-crap-out", you have to question yourself "can I get any use of this data?", if you can, then most likely the Analyser can too, if not, then do not expect any miracle from Analyser.

**Comment by JuliaNaumetsss** [comment]
@HardNorth, thanks for the reply!
I have additional question: Is it possible to save/take 'history of actions' information of the first runs of the launch? 
![image](https://github.com/user-attachments/assets/72bdd6ad-4444-481e-adb7-8daae7f11f93)

Right now we have information only on 'retry' run:
![image](https://github.com/user-attachments/assets/3a0b07c1-35b0-426c-aa43-7fe3536eebc2)

**Comment by HardNorth** [comment]
@JuliaNaumetsss No, unfortunately. This is just not implemented, I believe. Actually, original Item is rewritten on retry.
